applications/data_analysis/data_analysis.rst

```diff
@@ -8,3 +8,4 @@ Data Analysis
 .. toctree::
    :maxdepth: 1
    :hidden:
+   spectral_analysis
```

Comment:
Missing the `ipynb` extension, which I think is why it is not being included on the sidebar